src/Integration/Binding/RuleSetReferenceChecker.cs

```diff
@@ -22,31 +22,60 @@
 using System.Diagnostics;
 using System.IO;
 using System.Linq;
+using EnvDTE;
 using Microsoft.VisualStudio.CodeAnalysis.RuleSets;
 
-namespace SonarLint.VisualStudio.Integration
+namespace SonarLint.VisualStudio.Integration.Binding
 {
-    internal static class RuleSetIncludeChecker
+    internal class RuleSetReferenceChecker : IRuleSetReferenceChecker
```

Comment:
I wanted to re-use this logic in CSharpVBBindingOperation so I extracted it to class.